x-pack/plugins/aiops/public/components/log_categorization/show_flyout.tsx

```diff
@@ -53,34 +54,32 @@ export async function showCategorizeFlyout(
       };
       const datePickerDeps = {
         ...pick(appDependencies, ['data', 'http', 'notifications', 'theme', 'uiSettings']),
-        toMountPoint,
+        toMountPoint: toMountPointDeprecated,
```

Comment:
should we also update the date picker?

Reply:
yes, I didn't realise it was a in a package owned by ML. I assumed it was a core component.

Reply:
Updated in 9b8e1e65e7960dc72524e2b8b8023c41dd9472d2